packages/renovate/project.bri

```diff
@@ -15,11 +15,11 @@ const source = Brioche.gitCheckout({
   ref: project.version,
 });
 
-// Pin to Node.js v22
-const nodejs = nodejsRecipe({ version: "22" });
-const pnpm = pnpmRecipe({ nodejs });
-
 export default function renovate(): std.Recipe<std.Directory> {
+  // Pin to Node.js v22
+  const nodejs = nodejsRecipe({ version: "22" });
```

Comment:
This package does still not support newer version of Node.js:

```
WARN Unsupported engine: wanted: {"node":"^22.13.0"} (current: {"node":"v24.10.0","pnpm":"10.19.0"})
```

See https://github.com/renovatebot/renovate/pull/38940 and https://github.com/renovatebot/renovate/pull/38939